pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -40,6 +45,18 @@ func (r *ReconcilePerconaServerMongoDB) reconcileCluster(ctx context.Context, cr
 		return api.AppStateInit, nil
 	}
 
+	if cr.Spec.MultiCluster.Enabled {
+		for _, pod := range pods.Items {
+			imported, err := isServiceImported(ctx, r.client, cr, &pod)
+			if err != nil {
+				return api.AppStateError, errors.Wrapf(err, "check if service is imported for %s", pod.Name)
+			}
+			if !imported {
+				return api.AppStateInit, nil
+			}
```

Comment:
Since service imports may be created after a few minutes, I believe we should add a log:
```suggestion
			if !imported {
			         log.Info("waiting for service import", "replset", replset.Name, "pod", pod.Name)
				return api.AppStateInit, nil
			}
```